README.md

```diff
@@ -1,150 +1,3 @@
-# ALC Kubernetes Spark
-A kubernetes cluster configured to run spark jobs
+# Piezo
+A tool to run spark jobs on Kubernetes
```

Comment:
`Spark` should begin with a capital C. Please point readers to the [wiki](https://github.com/ukaea/piezo/wiki)